apps/backend/src/libs/modules/encryption/libs/types/encryption-result.type.ts

```diff
@@ -0,0 +1,6 @@
+type EncryptionResult = {
```

Comment:
usually we name such types as  `<method-name>Result`, don't forget about file name
```suggestion
type EncryptResult = {
```